src/ffi/mod.rs

```diff
@@ -36,6 +36,9 @@ pub use self::lua::{
     lua_Unsigned, lua_Writer,
 };
 
+// Constants
+pub use self::lua::LUA_SIGNATURE;
+
```

Comment:
For constants there is a dedicated section under comment `constants from lua.h`. This particular constant seems not in use and could be not imported.